chart/values.yaml

```diff
@@ -7,21 +7,21 @@ replicaCount: 1
 elasticsearch:
 # if `enabled = false` you need to set search.host
   enabled: true
-  imageTag: 6.8.6
+  imageTag: 6.8.8
```

Comment:
This is already something that probably shouldn't be in here - since this is the 'official' chart, we shouldn't be using a version of ES that isn't tested. Can you remove this change?